app/jobs/set_participant_start_date_job.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# This is a one off job to update all of the induction start dates from DQT for
+# all of the 2021/2022 ECTs that do not currently have them.
+# It runs in batches of 200 as we have a 300 lookup per minute limit on the API
+# There are approx 57600 participants in this state currently
+class SetParticipantStartDateJob < ApplicationJob
+  def perform
+    ParticipantProfile::ECT
+      .joins(schedule: :cohort)
+      .eligible_status
+      .where(cohort: { start_year: [2021, 2022] })
+      .where(induction_start_date: nil)
+      .order(:created_at)
+      .limit(200)
+      .each do |participant_profile|
+        ActiveRecord::Base.no_touching do
+          Participants::SetStartDateFromDQT.call(participant_profile:)
+        end
+      end
+  rescue StandardError => e
+    Rails.logger.error("SetParticipantStartDateJob: #{e.message}")
+  end
+end
```

Comment:
I think this job is bypassing the "setup permanent cohort" even for participants registered since June?!

Reply:
Yes that's a good point I hadn't consider that they might've said they were late 2022 cohort ones but actually turned out to be 2023, I'll add that as a condition 👍

Reply:
@peteryates The so called `Participants::SyncDqtInductionStartDate` actually calls `Induction::AmendParticipantCohort`  in case the participant needs their cohort amended once we get their induction start date from the AB via DQT